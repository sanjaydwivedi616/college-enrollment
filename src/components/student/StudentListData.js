import React from 'react'

const StudentListData = (props) =>{
    return (
        <table className="table">
            <thead>
                <tr>
                    <th>Sl.No</th>
                    <th>First Name</th>
                    <th>Last Name</th>
                    <th>Email</th>
                    <th>Contact</th>
                    <th>10%</th>
                    <th>12%</th>
                    <th>Course</th>
                    <th>Status</th>
                </tr>
            </thead>
            <tbody>
                {props.children}
            </tbody>

        </table>
    )
}

export default StudentListData;