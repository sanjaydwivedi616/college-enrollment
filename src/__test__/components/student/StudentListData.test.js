import StudentListData from "../../../components/student/StudentListData";
import renderer from 'react-test-renderer';
import { mount, shallow} from "enzyme";


describe('Whene student list is render', () => {

    it('Whene list data is render html.', () => {
        const dataListText = renderer.create(<StudentListData />)
        expect(dataListText).toMatchSnapshot();
    })
    /* 
    it('should render header list', () => {
        const wrapper = shallow(<StudentListData />);
        expect(wrapper.find('.table').length).toBe(1);
      }); */
})