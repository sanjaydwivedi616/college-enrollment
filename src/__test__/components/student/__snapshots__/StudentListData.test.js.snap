// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Whene student list is render Whene list data is render html. 1`] = `
<table
  className="table"
>
  <thead>
    <tr>
      <th>
        Sl.No
      </th>
      <th>
        First Name
      </th>
      <th>
        Last Name
      </th>
      <th>
        Email
      </th>
      <th>
        Contact
      </th>
      <th>
        10%
      </th>
      <th>
        12%
      </th>
      <th>
        Course
      </th>
      <th>
        Status
      </th>
    </tr>
  </thead>
  <tbody />
</table>
`;
